[2025-01-26T09:27:24.251+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T09:27:24.456+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_ebooks.create_table manual__2025-01-26T09:26:13.458326+00:00 [queued]>
[2025-01-26T09:27:24.513+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_ebooks.create_table manual__2025-01-26T09:26:13.458326+00:00 [queued]>
[2025-01-26T09:27:24.521+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-26T09:27:24.786+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_table> on 2025-01-26 09:26:13.458326+00:00
[2025-01-26T09:27:25.128+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_ebooks', 'create_table', 'manual__2025-01-26T09:26:13.458326+00:00', '--job-id', '1068', '--raw', '--subdir', 'DAGS_FOLDER/app.py', '--cfg-path', '/tmp/tmpazwcuvjr']
[2025-01-26T09:27:25.183+0000] {standard_task_runner.py:105} INFO - Job 1068: Subtask create_table
[2025-01-26T09:27:25.133+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4365) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T09:27:25.245+0000] {standard_task_runner.py:72} INFO - Started process 4405 to run task
[2025-01-26T09:27:26.547+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_ebooks.create_table manual__2025-01-26T09:26:13.458326+00:00 [running]> on host 34edd304782d
[2025-01-26T09:27:28.915+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_ebooks' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T09:26:13.458326+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-26T09:26:13.458326+00:00'
[2025-01-26T09:27:28.921+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-26T09:27:28.929+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-26T09:27:28.931+0000] {logging_mixin.py:190} INFO - Current task name:create_table state:running start_date:2025-01-26 09:27:24.458394+00:00
[2025-01-26T09:27:28.940+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_ebooks and current dag run status:running
[2025-01-26T09:27:28.950+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T09:27:28.962+0000] {sql.py:278} INFO - Executing: 
    CREATE TABLE IF NOT EXISTS public.ebooks (
        id SERIAL PRIMARY KEY,
        judul TEXT NOT NULL,
        penulis TEXT,
        link TEXT
    );
    
[2025-01-26T09:27:29.097+0000] {base.py:84} INFO - Retrieving connection 'ebooks_connection'
[2025-01-26T09:27:29.173+0000] {base.py:84} INFO - Retrieving connection 'ebooks_connection'
[2025-01-26T09:27:29.632+0000] {sql.py:553} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS public.ebooks (
        id SERIAL PRIMARY KEY,
        judul TEXT NOT NULL,
        penulis TEXT,
        link TEXT
    );
    , parameters: None
[2025-01-26T09:27:29.734+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T09:27:29.751+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_ebooks, task_id=create_table, run_id=manual__2025-01-26T09:26:13.458326+00:00, execution_date=20250126T092613, start_date=20250126T092724, end_date=20250126T092729
[2025-01-26T09:27:29.839+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-26T09:27:29.860+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-26T09:27:29.862+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_ebooks queued_at:2025-01-26 09:26:13.529511+00:00
[2025-01-26T09:27:29.864+0000] {logging_mixin.py:190} INFO - Task hostname:34edd304782d operator:PostgresOperator
[2025-01-26T09:27:29.996+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T09:27:30.147+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-26T09:27:30.177+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-26T09:27:30.269+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
